index.js

```diff
@@ -38,18 +51,40 @@ function loadSpreadsheet(options, cb) {
             if (err) {
                 return cb(err)
             }
-            console.log(row_data[0].value, row_data[1].value)
             var converted = {};
             var langs = [];
+            var commentsColumnIndex;
+
             for (var i = 1; i < colCount; i++) {
-                langs[i] = row_data[i].value;
+                if (options.ignoreCommentsColumn == true && row_data[i].value == 'comments') {
```

Comment:
it's minor
do you think, hardcode column name is good idea?
maybe instead of `options.ignoreCommentsColumn`
just specify `options.ignoreCommentsColumnIndex`

Reply:
I think hardcoding it is a good idea because otherwise people will keep having to juggle the column index... if people follow a standard format then nobody needs to enter in indexes anywhere, it also encourages everyone to keep a clean sheet and any work in progress is done in another sheet

Reply:
ok, I do not have strong arguments against it, so let keep this options